test/DurableTask.SqlServer.Tests/Integration/DatabaseManagement.cs

```diff
@@ -490,6 +495,16 @@ static void ValidateDatabaseSchema(TestDatabase database, string schemaName = "d
             }
 
             Assert.Empty(expectedFunctionNames);
+
+            // Verify that the schema version in the database matches the expected version
+            // Note that we'll need to change the expected version here whenever we introduce new schema.
+            SemanticVersion currentSchemaVersion = await SharedTestHelpers.GetCurrentSchemaVersionAsync(
+                this.output,
+                database.ConnectionString,
+                schemaName);
+            Assert.Equal(1, currentSchemaVersion.Major);
+            Assert.Equal(2, currentSchemaVersion.Minor);
+            Assert.Equal(0, currentSchemaVersion.Patch);
```

Comment:
This isn't ideal, but I wasn't able to think of a better process. The version is supposed to match the nuget package version, but we currently only update the schema version when we actually make changes, so this number is going to always be equal or less than the nuget package version.

My expectation, however, is that the next time we update the schema, the schema version will be updated automatically, and this test will fail. This failure will alert us to update this value, and the person updating it will be able to make sure that the database schema version matches what we intuitively expect. People will be forced to update this in order to get the test passing again.